sidebars.js

```diff
@@ -47,6 +47,13 @@ module.exports = {
         'core-functionality/mnv-recomposition',
         'core-functionality/variant-ids'
       ],
+    },
+	{
+      type: 'category',
+      label: 'Utilities',
+      items: [
+        'utilities/jasix'
+      ],
     },
   ],
 };
```

Comment:
Please add content about the JASIX file format and how indexing works. There have been a few people that are interested in using the .jsi files in their own tools.